intera_examples/scripts/go_to_cartesian_pose.py

```diff
@@ -86,6 +86,9 @@ def main():
     parser.add_argument(
         "--rotational_accel", type=float, default=1.57,
         help="The max rotational acceleration of the endpoint (rad/s/s)")
+    parser.add_argument(
+        "--timeout", type=float, default=None,
+        help="Max time allocated to finish the goal before returning (sec). None is interpreted as an infinite timeout.")
```

Comment:
Small suggestion:
`Max time in seconds to complete motion goal before returning. None is interpreted as an infinite timeout.`